rubycritic.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'flog', '4.3.2'
   spec.add_runtime_dependency 'reek', '3.11'
   spec.add_runtime_dependency 'parser', '~> 2.3'
+  spec.add_runtime_dependency 'ruby_parser', '>= 3.8'
```

Comment:
But with this set up we would also pull in v4 which might be api breaking.
How about:

```Ruby
spec.add_runtime_dependency 'ruby_parser', [ '>= 3.8', ~> '3.8' ]
```

?